tests/test_work_unit.py

```diff
@@ -79,9 +79,9 @@ def test_save_and_load_fits(self):
 
```

Comment:
Can you add a check that the times match? I don't think work_unit.py is setting the obstime.